packages/slice-machine/src/models/slice/reducer.ts

```diff
@@ -85,7 +85,6 @@ export function reducer(
         return {
           ...prevState,
           screenshotUrls: screenshots,
-          infos: prevState.infos,
```

Comment:
what is that ?

Reply:
It's a property that is already passed inside the spread 2 line above :)
I just wanted to simplify it.

Reply:
the type of it is "componentInfos"

Reply:
ok, thks for the information